style-guide/expressions.md

```diff
@@ -118,7 +117,7 @@ fn main() {
 ### Closures
 
 Don't put any extra spaces before the first `|` (unless the closure is prefixed
-by `move`), but put a space between the second `|` and the expression of the
+by `move`); put a space between the second `|` and the expression of the
```

Comment:
```
let x = |a| a; // a space before |
test( |a| a ); // a space before |
```
which one is `extra space`?

Reply:
The latter. The former is covered by always putting spaces around `=`.